src/many-abci/src/module.rs

```diff
@@ -102,6 +119,79 @@ fn _tm_query_from_many_filter(
     Ok(query)
 }
 
+fn transform_list_args(
+    blockchain::ListArgs {
+        count,
+        order,
+        filter,
+    }: blockchain::ListArgs,
+) -> Result<(u64, Order, Query), ManyError> {
```

Comment:
`count` should be a `u8` and not a `u64`

Reply:
I deferred the downcast until later in the chain to simplify some of the intermediate calculations using `count`.